microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -23,18 +25,20 @@ To turn on \(or off\) experimental features in Microsoft Edge, complete the foll
     *   Select `Control`+`Shift`+`I` \(Windows, Linux\) or `Command`+`Option`+`I` \(macOS\).  For more information, navigate to [Microsoft Edge DevTools keyboard shortcuts][DevtoolsShortcutsIndex].  
 1.  Open the [Settings][DevToolsCustomizeIndexSettings] pane.  
     *   Select `Shift`+`?`.  For more information, navigate to [Microsoft Edge DevTools keyboard shortcuts][DevtoolsShortcutsIndex].  
-1.  On the left side of the **Settings** pane, choose the **Experiments** section.  
+1.  On the left side of the **Settings** pane, select the **Experiments** section.  
     
     :::image type="complex" source="../media/experiments-devtools.msft.png" alt-text="The Experiments page in Settings" lightbox="../media/experiments-devtools.msft.png":::
        The **Experiments** page in **Settings**  
     :::image-end:::  
     
-1.  On the **Experiments** page, scroll through the list of all available experimental features and choose the checkbox next to each feature that you want to test.  
+1.  On the **Experiments** page, scroll through the list of all available experimental features and select the checkbox next to each feature that you want to test.  
 1.  Close and reopen Microsoft Edge DevTools.  
     
 > [!NOTE]
-> Experimental features are constantly being updated and may cause performance issues.  To turn off an experimental feature, open the **Experiments** page and clear the checkbox of the experimental feature that you want to turn off.  
+> Experimental features are constantly being updated and might cause performance issues.  To turn off an experimental feature, open the **Experiments** page and clear the checkbox of the experimental feature that you want to turn off.  
+
 
+<!-- ====================================================================== -->
 ## Highlighted experimental features  
```

Comment:
This section should be deleted now that each item is an h2 and appears in the In This Article nav.